app/models/spree/gateway/flow_io.rb

```diff
@@ -102,6 +104,15 @@ def request_refund_store_result(order, amount)
         end
       end
 
+      def schedule_status_check(response, order)
+        return unless response.captures.any? { |cap| cap.capture.status.value != 'succeeded' }
```

Comment:
you can do something like 

```
return if (captures = response.captures.any? { |cap| cap.capture.status.value != 'succeeded' }).blank?
```

and then avoid duplicating the call to populate the `captures` variable in line 110

Reply:
Updated!